CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Tamr VM Terraform Module
 
-## v4.3.0 - December 16th 2021
+
+## v4.4.0 - December 20th 2021
```

Comment:
Thinking about the upgrade path from 4.3.0 to 4.4.0, since `additional_policy_arns` is required this should be a major version.

Going from the previous version will require they add this field to do the upgrade, even if they keep it empty to continue using the deprecated variable.

General guidance on versions:
Major -> introduces new features that do require changes to the user's terraform beyond the version number
Minor -> introduces new features that do not require any changes to the user's terraform
Patch -> bug fixes that do not require any changes to the user's terraform


That said, if we wanted to keep this a minor release, we could add a default to `additional_policy_arns`, but I'm ok with this being a major version, to help guide users of the module to the new variable.

Reply:
In order to keep this a minor release I added a default to 'additional_policy_arns'.